main/webpack.config.js

```diff
@@ -18,8 +18,8 @@ module.exports = {
     }),
     new HtmlWebpackPlugin({
       template: 'index.html',
-      filename: 'ol.html',
-      chunks: ['ol'],
+      filename: 'openlayers.html',
+      chunks: ['openlayers'],
```

Comment:
I think if you use `openlayers` here, you also need to rename the entry point. Anyway it would be good to rename it:
```
   entry: {
     mapbox: './src/mapbox.js',
     openlayers: './src/openlayers.js',
   },
```
(and also rename the .js files accordingly)